bin/tornadovm-installer

```diff
@@ -30,8 +31,8 @@ tornadoReq.check_python_dependencies()
 import wget
 import installer_config as config
 
-__DIRECTORY_DEPENDENCIES__ = "etc/dependencies"
-__VERSION__ = "v1.0.1-dev"
+__DIRECTORY_DEPENDENCIES__ = os.path.join("etc", "dependencies")
+__VERSION__ = "v1.0.3"
```

Comment:
Good catch. Actually, it is the `v0.1.0.4-dev`. Let's change it to that.

Reply:
I set it to `v1.0.4-dev`. Your value `v0.1.0.4-dev` ist a typo, right? (note the 0 after the v).

Reply:
Yes, sorry. it was a typo.